src/test/java/org/apache/maven/shared/utils/io/SymlinkTestSetup.java

```diff
@@ -36,18 +37,18 @@ public static File createStandardSymlinkTestDir( File root )
         srcDir.mkdirs();
         File target = new File( srcDir, "targetDir" );
         target.mkdirs();
-        write( new File( target, "targetFile.txt" ), "a regular File payload" );
+        write( new File( target, "targetFile.txt" ), "a regular File payload", StandardCharsets.UTF_8 );
```

Comment:
I like using static imports because here in this example the line would be short in GH without breaking to new line.

Reply:
It's still within Maven's (rather large) 120 character column standard. I like to know where my constants are coming from. This isn't the only UTF_8 constant floating around in Javaland.

Reply:
In general I prefer static imports for methods. For a constant like this I like to include the classname.
It is a matter of taste, I'm fine with this change.